### ISSUE TITLE
Serial.println cannot output to the serial monitor

### ISSUE THREAD

**Issue by iamxiaojianzheng** [HomeSpan/HomeSpan#640]
After my test, in the nio program that uses HomeSpan, Serial.println cannot be output. 
But in other nio programs that don't use HomeSpan, everything works fine with Serial.println.

**Comment by HomeSpan** [comment]
What is "nio"?

**Comment by iamxiaojianzheng** [comment]
> What is "nio"?

Sorry, is ino.

**Comment by HomeSpan** [comment]
Okay, so within the Arduino IDE.  Is this the same as #639 (in which case see my response about baud settings).

**Comment by iamxiaojianzheng** [comment]
I'm using the code from #639 , but replacing Serial.println with printf, because my attempts to use Serial.println didn't work.

but other code is ok. for example:

```nio
#include <FastLED.h>

#define NUM_LEDS 1
#define DATA_PIN 10

CRGB leds[NUM_LEDS];

void setup() {
  Serial.begin(115200);
  // put your setup code here, to run once:
  FastLED.addLeds<WS2812, DATA_PIN, GRB>(leds, NUM_LEDS);  //RGB ou GRB ??
  Serial.println("setup");
}

void loop() {
  leds[0] = CRGB::Red;
  FastLED.show();
  Serial.println("red");
  delay(500);
  leds[0] = CRGB::Green;
  FastLED.show();
  Serial.println("green");
  delay(500);
  leds[0] = CRGB::Blue;
  FastLED.show();
  Serial.println("blue");
  delay(500);
  leds[0] = CRGB::Black;
  FastLED.show();
  delay(500);
}
```

**Comment by HomeSpan** [comment]
Can this issue be closed out - I think I answered in #639?
